runtime/Js.res

```diff
@@ -190,7 +190,7 @@ type nullable<+'a> = Js_null_undefined.t<'a> = Value('a) | @as(null) Null | @as(
 type null_undefined<+'a> = nullable<'a>
 
 external toOption: nullable<'a> => option<'a> = "%nullable_to_opt"
-external undefinedToOption: undefined<'a> => option<'a> = "%undefined_to_opt"
+external undefinedToOption: undefined<'a> => option<'a> = "%identity"
```

Comment:
This is an observable behaviour change, need to use an existing conversion function (now that the primitive does not exist anymore).